package.json

```diff
@@ -335,6 +335,11 @@
 				"command": "tabnine.accept-inline-suggestion",
 				"when": "tabnine.snippet-suggestion:enabled && tabnine.in-inline-suggestions || tabnine.inline-suggestion:enabled && tabnine.in-inline-suggestions"
 			},
+			{
+				"key": "tab",
+				"command": "tabnine.tab-override",
+				"when": "tabnine.tab-override && suggestWidgetVisible && textInputFocus"
+			},
```

Comment:
how does it play with the other "tab" key we're defining? can there be a situation where both key overrides are active? what happens if so?

Reply:
each override should be guarded with the relevant context flag. this override is enabled only if `tabnine.tab-override` context flag is enabled.